newrelic_lambda_cli/templates/import-template.yaml

```diff
@@ -58,8 +61,8 @@ Resources:
     DeletionPolicy: Delete
     Properties:
       CodeUri:
-        Bucket: awsserverlessrepo-changesets-1iiv3xa62ln3m
-        Key: 466768951184/arn:aws:serverlessrepo:us-east-1:463657938898:applications-NewRelic-log-ingestion-versions-2.2.1/2dcb4087-186a-42c8-bc1f-cf93780786c0
+        Bucket: awsserverlessrepo-changesets-18ssd5swmy82n
```

Comment:
Why is this changing?

Reply:
Can exclude this from final PR but I needed to deploy my lambda from an eu-west-1 bucket... need to go back to using `get_sar_template_url()` which generates the bucket values for new installs.

Reply:
I removed this commit from the PR.